### PR TITLE
moved changes to main

### PR DESCRIPTION
**Pull request by beemyfriend**
This is my first attempt at the website, I still need to get a evenbrite page activated, but I want to confirm costs before I do that. Can you also check to see if the email/contact feature works?

### CODE REVIEW

File: config.toml [hunk 1]
```diff
@@ -1,8 +1,8 @@
-# Site settings
-baseurl = "https://satrdays-event-template.netlify.com/"
+baseURL = "http://example.org/"
 languageCode = "en-us"
-title = "satRday"
-theme = ["hugo-satrdays-theme", "hugo-agency-theme"]
+title = "My New Hugo Site"
```
Comment (DaveParr) [inline config.toml]: need to change back to old title

**Comment by DaveParr** [comment]
after that you can probably merge it yourself, then I'll check the email link for you :)

**Comment by beemyfriend** [comment]
I changed the base url and the title. I'm not sure if there are any other issues. I don' t think I have write access to this repository so I cannot merge it myself.

**Comment by stephlocke** [comment]
@beemyfriend write access granted

**Comment by stephlocke** [comment]
Looking great Ben! https://dc2018.satrdays.org/

**Pull request merged**